pkg/apis/batch/validation/validation_test.go

```diff
@@ -1349,6 +1373,46 @@ func TestValidateJobUpdate(t *testing.T) {
 				job.Spec.ManualSelector = pointer.Bool(true)
 			},
 		},
+		"invalid attempt to set managedBy field": {
+			old: batch.Job{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "abc",
+					Namespace: metav1.NamespaceDefault,
+					Labels:    map[string]string{},
+				},
+				Spec: batch.JobSpec{
+					Selector: validGeneratedSelector,
+					Template: validPodTemplateSpecForGenerated,
+				},
+			},
+			update: func(job *batch.Job) {
+				job.Spec.ManagedBy = ptr.To("custom-controller")
+			},
+			err: &field.Error{
+				Type:  field.ErrorTypeInvalid,
+				Field: "spec.managedBy",
+			},
+		},
+		"invalid update of the managedBy field": {
+			old: batch.Job{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "abc",
+					Namespace: metav1.NamespaceDefault,
```

Comment:
```suggestion
				ObjectMeta: validJobObjectMeta,
```

Reply:
done, use the oneliner as in other test cases